build.gradle

```diff
@@ -442,6 +442,7 @@ subprojects {
                  'opencensus-exporter-stats-signalfx',
                  'opencensus-exporter-stats-stackdriver',
                  'opencensus-exporter-trace-datadog',
+                 'opencensus-exporter-trace-elasticsearch',
```

Comment:
Please also add this to CHANGELOG.md and https://github.com/census-instrumentation/opencensus-java/blob/master/RELEASING.md#update-release-versions-in-documentations-and-build-files.